src/controllers/albumController.js

```diff
@@ -161,6 +161,15 @@ const getAlbumWithAllUsersWhoPurchasedIt = async (req, res) => {
 
 const getFilteredAlbums = async (req, res) => {
   const { limit, order, offset, albumName, artistName } = req.query;
+
+  console.log('reqoriginalUrl is', req.originalUrl);
+
+  const url = new URL('http://localhost:8000' + req.originalUrl); // create a "URL" object
+  //Akos: req.originalUrl solo gives an error
+
+  const parsedOffset = parseInt(offset, 10);
+  const parsedLimit = parseInt(limit, 10);
```

Comment:
to save having to do the comparisons `parsedOffset || 0` and `parsedLimit || 10` all throughout the code, I _think_ we can clean things up by doing that just one time here:

```suggestion
  const parsedOffset = parseInt(offset, 10) || 0;
  const parsedLimit = parseInt(limit, 10) || 10;
```